### ISSUE TITLE
[dev.icinga.com #9994] Deadlock found when trying to get lock; try restarting transaction Icinga v2.3.8 MySQL 5.6.24 schema version: '1.13.0'

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1952]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/9994

**Created by frjaraur on 2015-08-25 16:51:36 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2016-10-05 11:23:15 +00:00_)
Target Version: _(none)_
Last Update: _2016-10-05 11:23:15 +00:00 (in Redmine)_

---

I have lots of "Deadlock found when trying to get lock; try restarting transaction" in my icinga logfile.
For example..
\[2015-08-25 18:38:36 +0200\] critical/IdoMysqlConnection: Error "Deadlock found when trying to get lock; try restarting transaction" when executing query "UPDATE icinga\_hoststatus SET acknowledgement\_type = '0', active\_checks\_enabled = '1', check\_command = 'hostalive', check\_source = 'mgmt05', check\_type = '0', current\_check\_attempt = '1', current\_notification\_number = '0', current\_state = '0', endpoint\_object\_id = 78, event\_handler = '', event\_handler\_enabled = '1', execution\_time = '4.0203409194946289', flap\_detection\_enabled = '0', has\_been\_checked = '1', host\_object\_id = 3032, instance\_id = 1, is\_flapping = '0', is\_reachable = '1', last\_check = FROM\_UNIXTIME(1440520508), last\_hard\_state = '0', last\_hard\_state\_change = FROM\_UNIXTIME(1438849846), last\_state\_change = FROM\_UNIXTIME(1438849846), last\_time\_down = FROM\_UNIXTIME(1438849805), last\_time\_up = FROM\_UNIXTIME(1440520534), latency = '0', long\_output = '', max\_check\_attempts = '1', modified\_host\_attributes = '0', next\_check = FROM\_UNIXTIME(1440520566), normal\_check\_interval = '1', notifications\_enabled = '1', output = 'PING OK - Packet loss = 0, RTA = 1.02 ms', passive\_checks\_enabled = '1', percent\_state\_change = '0', perfdata = 'rta=1.025000ms;3000.000000;5000.000000;0.000000 pl=0;80;100;0', problem\_has\_been\_acknowledged = '0', process\_performance\_data = '1', retry\_check\_interval = '0.5', scheduled\_downtime\_depth = '0', should\_be\_scheduled = '1', state\_type = '1', status\_update\_time = FROM\_UNIXTIME(1440520716) WHERE host\_object\_id = 3032"
Context:
(0) Reconnecting to MySQL IDO database 'ido-mysql'

\[2015-08-25 18:38:36 +0200\] critical/IdoMysqlConnection: Exception during database operation: Verify that your database is operational!
\[2015-08-25 18:38:36 +0200\] information/IdoMysqlConnection: MySQL IDO instance id: 1 (schema version: '1.13.0')

My Icinga installation data:
Icinga v2.3.8
mysql 5.6.24

Don't know I it is a new issue or is realted to [\#8835,](https://dev.icinga.com/issues/8835) so I update and open this issue on tracker.

Is there anything else I could review?.

Many Thanks,
Javier R.

---

**Parent Task:** [#8612](https://dev.icinga.com/issues/8612)
